src/backend/distributed/commands/distribute_object_ops.c

```diff
@@ -1512,6 +1546,23 @@ GetDistributeObjectOps(Node *node)
 					return &Trigger_Rename;
 				}
 
+				case OBJECT_VIEW:
+				{
+					return &View_Rename;
+				}
+
+				case OBJECT_COLUMN:
```

Comment:
This is a little confusing for me to follow. why we need this block, does pg support rename column of a view? Maybe, can you remind me which command triggers this code-path?

Reply:
e.g., why do we need the `else` block, it didn't exists before this PR?

Reply:
Yes, PG supports altering the name of the column of a view. We've added that check to route alter column name for views and tables properly. Default case has been added to not get compile time warning.

Reply:
Yes, PG supports altering the name of the column of a view. We've added that check to route alter column name for views and tables properly. Default case has been added to not get compile time warning.